locales/en.lua

```diff
@@ -6,6 +6,7 @@ local Translations = {
         not_correct_type = "You can't store this type of vehicle here",
         not_enough = "Not enough money",
         no_garage = "None",
+        not_enough_space = "Someting blocking the space!"
```

Comment:
Should be "Something blocking the space!"

Reply:
Fixed thanks!